### PR TITLE
remove unnecessary port fields in frontend_key

### LINKED ISSUE #277: frontend data structure adjustment in the kmesh workload mode
**What would you like to be added**:
1.the name of service_id variable in frontend value has limitations and needs to be renamed;
2.the port variable in the frontend key seems to be redundant and needs to be removed;
3.docs of workload mode need to be refreshed.
Plan to handle the above issues in three separate PR
**Why is this needed**:

### PR DESCRIPTION
**Pull request by kwb0523**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #277

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by hzxuzhonghu — COMMENTED** [review]
I cannot understand how we do lb based only on the target ip now

**Comment by kwb0523** [comment]
> I cannot understand how we do lb based only on the target ip now



> I cannot understand how we do lb based only on the target ip now

update the workload proposal later.

**Review by hzxuzhonghu — DISMISSED** [review]
Hope to see the following fix on direct pod access soon

File: bpf/kmesh/workload/include/ctx/sock_addr.h [hunk 1]
```diff
@@ -29,8 +29,7 @@ typedef struct bpf_sock_addr ctx_buff_t;
 
 #define DECLARE_VAR_ADDRESS(ctx, name)                                                                                 \
```
[the remark this answers is not in the document]
Comment (kwb0523) [reply bpf/kmesh/workload/include/ctx/sock_addr.h]: fixed

**Comment by kwb0523** [comment]
> Hope to see the following fix on direct pod access soon

okay, asap

**Comment by kwb0523** [comment]
> Hope to see the following fix on direct pod access soon

okay, asap

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**